src/unaudited/ECDSAServiceManagerBase.sol

```diff
@@ -106,6 +106,21 @@ abstract contract ECDSAServiceManagerBase is
         _createAVSRewardsSubmission(rewardsSubmissions);
     }
 
+    /// @inheritdoc IServiceManager
+    function createAVSPerformanceRewardsSubmission(
+        IRewardsCoordinator.PerformanceRewardsSubmission[]
+            calldata performanceRewardsSubmissions
+    ) external virtual onlyRewardsInitiator {
+        _createAVSPerformanceRewardsSubmission(performanceRewardsSubmissions);
+    }
+
+    /// @inheritdoc IServiceManager
+    function setClaimerFor(
+        address claimer
+    ) public virtual onlyRewardsInitiator {
```

Comment:
same here for `onlyOwner`

Reply:
Done in [3d99748](https://github.com/Layr-Labs/eigenlayer-middleware/pull/315/commits/3d9974848612772ac69b30b703648497e9c11bff)